usr/local/www/system_advanced_sysctl.php

```diff
@@ -135,155 +135,93 @@
-							<tr>
-								<td colspan="2" valign="top" class="listtopic"><?=gettext("Edit system tunable"); ?></td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top" class="vncellreq"><?=gettext("Tunable"); ?></td>
-								<td width="78%" class="vtable">
-									<input size="65" name="tunable" value="<?php echo $pconfig['tunable']; ?>" />
-								</td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top" class="vncellreq"><?=gettext("Description"); ?></td>
-								<td width="78%" class="vtable">
-									<textarea rows="7" cols="50" name="descr"><?php echo $pconfig['descr']; ?></textarea>
-								</td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top" class="vncellreq"><?=gettext("Value"); ?></td>
-								<td width="78%" class="vtable">
-									<input size="65" name="value" value="<?php echo $pconfig['value']; ?>" />
-								</td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top">&nbsp;</td>
-								<td width="78%">
-									<input id="submit" name="Submit" type="submit" class="formbtn" value="<?=gettext("Save"); ?>" />
-									<input type="button" class="formbtn" value="<?=gettext("Cancel");?>" onclick="window.location.href='<?=$referer;?>'" />
-									<?php if (isset($id) && $a_tunable[$id]): ?>
-									<input name="id" type="hidden" value="<?=htmlspecialchars($id);?>" />
-									<?php endif; ?>
-								</td>
-							</tr>
-						</table>
-					</form>
-				</div>
-			</td>
-		</tr>
-		<?php endif; ?>
-	</table>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+	</thead>
+	<?php foreach ($tunables as $i => $tunable):
+		if (!isset($tunable['modified']))
+			$i = $tunable['tunable'];
+	?>
+	<tr>
+		<td ondblclick="document.location='system_advanced_sysctl.php?act=edit&amp;id=<?=$i;?>';">
+			<?php echo $tunable['tunable']; ?>
+		</td>
+		<td ondblclick="document.location='system_advanced_sysctl.php?act=edit&amp;id=<?=$i;?>';">
+			<?php echo $tunable['descr']; ?>
+		</td>
+		<td ondblclick="document.location='system_advanced_sysctl.php?act=edit&amp;id=<?=$i;?>';">
+			<?php echo $tunable['value']; ?>
+			<?php
+				if($tunable['value'] == "default")
+					echo "(" . get_default_sysctl_value($tunable['tunable']) . ")";
+			?>
+		</td>
+		<td>
+			<a class="btn btn-xs btn-primary" href="system_advanced_sysctl.php?act=edit&amp;id=<?=$i;?>">edit</a>
+			<?php if (isset($tunable['modified'])): ?>
+			<a class="btn btn-xs btn-danger" href="system_advanced_sysctl.php?act=del&amp;id=<?=$i;?>" onclick="return confirm('<?=gettext("Do you really want to delete this entry?"); ?>')">Delete/Reset</a>
+			<?php endif; ?>
+		</td>
+	</tr>
+	<?php endforeach; unset($tunables); ?>
+	<tr>
+		<td></td>
+		<td></td>
+		<td></td>
+		<td>
+			<a class="btn btn-xs btn-primary" href="system_advanced_sysctl.php?act=edit">New</a>
+		</td>
+	</tr>
+</table>
+<?php else: ?>
+
+<form action="system_advanced_sysctl.php" method="post" name="iform" id="iform">
```

Comment:
I was going that route at first, but ran into some issues that would either require a logic/interface rewrite or a lot of work to the Form classes. Is there a reason that you're not using a php bootstrap framework? Like this: http://www.phpbootstrap.com/